src/components/AdminMarkerInfo/BoxInfo/BoxInfo.jsx

```diff
@@ -16,10 +16,28 @@ import { FYABackend } from '../../../common/utils';
 
 const BoxInfo = ({ selectedBox, setSelectedBox }) => {
   const [boxHistory, setBoxHistory] = useState([]);
+  const [boxHolderName, setBoxHolderName] = useState('Loading...');
```

Comment:
Maybe use a CONST loadingString = 'Loading..' and pass that in on the useStates. 
I know its a nitpick but might increase #performance by like 0.05 ms